File: nom-api/Nom.Orch/Models/Shopping/ShoppingListItemResponseModel.cs
// File: Nom.Orch/Models/Shopping/ShoppingListItemResponseModel.cs

namespace Nom.Orch.Models.Shopping
{
    public class ShoppingListItemResponseModel
    {
        public long Id { get; set; }
        public long ShoppingListId { get; set; }
        public string Name { get; set; } = string.Empty;
        public decimal? Quantity { get; set; }
        public bool IsCompleted { get; set; }
        public string? Note { get; set; }
        public long? IngredientId { get; set; }
        public long? RecipeId { get; set; }
        public int? Position { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime? ModifiedDate { get; set; }
    }
} 

File: nom-ui/src/app/core/models/shopping-list-item-response.model.ts
export interface ShoppingListItemResponse {
  id: number;
  shoppingListId: number;
  name: string;
  quantity: number | null;
  isCompleted: boolean;
  note: string | null;
  ingredientId: number | null;
  recipeId: number | null;
  position: number | null;
  createdDate: string;
  modifiedDate: string | null;
}
